packages/eslint-plugin-ui5-define-jsdocs/lib/utils/helper.js

```diff
@@ -0,0 +1,105 @@
+"use strict";
+
+const _ = require("lodash");
+const doctrine = require("doctrine");
+
+let createJsDocParamType = sModulePath => {
+  return _.join(_.split(sModulePath, "/"), ".");
+};
+
+let isRelevantModulePath = sModulePath => {
+  return sModulePath && sModulePath.match(/^[A-Z0-9]/i);
+};
+
+let getModuleArgumentsPairsToCheck = (aModules, aArguments) => {
+  const size = Math.min(_.size(aModules), _.size(aArguments));
+  const aElementAndParams = [];
+  _.forEach(_.take(aModules, size), (oElement, nIndex) => {
```

Comment:
consider two separate transformations, filter, map

Reply:
In this case it was not so useful using filter and map because we use index of an element